docs/cli/discount-code-generator.md

```diff
@@ -0,0 +1,153 @@
+# Discount Code Generator
+
+Generate unique discount codes to be imported to the [commercetools platform](https://dev.commercetools.com/). See usage below
+
+## Usage
+`npm install @commercetools/discount-code-generator --global`
+
+### CLI
+```
+Usage: discountCodeGenerator [options]
+
+Options:
+  --help, -h                 Show help text.                                                     [boolean]
+  --version, -v              Show version number.                                                [boolean]
+  --quantity, -q             Quantity of discount codes to generate. (Between 1 and 500000)     [required]
+  --code-length, -l          Length of the discount codes to generate.                       [default: 11]
+  --code-prefix, -p          Prefix for each code. No prefix will be used if omitted.        [default: ""]
+  --input, -i                Path to code options CSV or JSON file.
+  --output, -o               Path to store generated output file.                      [default: "stdout"]
+  --delimiter, -d            Used CSV delimiter for input and/or output file.               [default: ","]
+  --multivalueDelimiter, -m  Used CSV delimiter in multivalue fields for input/output file. [default: ";"]
+```
+
+#### Info on flags
+- The ```--quantity``` flag represents the number of codes to be generated. It must be a number between 1 and 500000
+- The ```--code-length``` flag is used to customise the discount codes according to individual use cases. (more info in Exmples)
```

Comment:
...little typo here... _Exmples_

also you can make _Examples_ an hyperlink to the correct section